trieste/acquisition/function/function.py

```diff
@@ -232,8 +227,28 @@ def __call__(self, x: TensorType) -> TensorType:
         return (self._eta - mean) * normal.cdf(self._eta) + variance * normal.prob(self._eta)
 
 
-class log_expected_improvement(AcquisitionFunctionClass):
-    def __init__(self, model: ProbabilisticModel, eta: TensorType):
+class LogExpectedImprovement(ExpectedImprovement):
+    """
+    Builder for the log expected improvement function.
+    """
+
+    def __init__(self, search_space: Optional[SearchSpace] = None):
+        """
+        :param search_space: The global search space over which the optimisation is defined. This is
+            only used to determine explicit constraints.
+        """
+        self._search_space = search_space
+        self._acq_function_cls = log_expected_improvement
+
+    def __repr__(self) -> str:
+        """"""
+        return f"LogExpectedImprovement({self._search_space!r})"
+
+
+class log_expected_improvement(expected_improvement):
+
+    @tf.function
+    def __call__(self, x: TensorType) -> TensorType:
```

Comment:
docsting below in the call should go to the class docstring I think, thats the actual builder this method is just evaluating it

Reply:
actually, it should probably go to LogExpectedImprovement, that's the one users will actually use and therefore read that doc